### DISCUSSION TITLE
ESP32-S2 DMA?

### DISCUSSION THREAD

**Discussion by supremeneuron** [Bodmer/TFT_eSPI#1451] [New processor/display support or capability request]
Bodmer,

    First, I want to thank you for a wonderful library!  Your recent inclusion of the ESP32-S2 has made one of my recent projects really pop.  Naturally, the library has the usually functionality but one element appears to be absent....It doesn't appear that the library for the ESP32-S2 currently has DMA capability.  Is this something that may appear in a future release of the library or is it that the ESP32-S2 chip is incapable of handling DMA to SPI for the ILI9341 and ILI9488 displays?

**Comment by Bodmer** [comment]
At the moment DMA does not work with the ESP32 S2 variant. I do not know why and the documentation is thin on DMA so this may not get resolved for some time.

**Reply by JohnWSatterfield** [reply]
Do we have an update on the DMA for ESP32 S2 variant?
